payments/models.py

```diff
@@ -2,7 +2,7 @@
 
 import json
 import logging
-from typing import Iterable
+from collections.abc import Iterable
```

Comment:
I think we should disable this rule. Applying the suggestion would affect eh result of generated documentation.

https://results.pre-commit.ci/run/github/951385/1734212786.rU_Pz5tETUKw6LwPmcu3HQ